### ISSUE TITLE
Relay to mailcow for repackaging and sending.

### ISSUE THREAD

**Issue by CameronMunroe** [mailcow/mailcow-dockerized#3021]
**Is your feature request related to a problem? Please describe.**

This is more a question if it is possible.

I have Server A and my Mailcow dockerized. I want to send mail from Server A to my Mailcow, have it repackage and send out. Currently attempts state that "gmail.com" is not an allowed host when setting Server A as a forwarding host. (( I initially read it as Relaying Hosts. )) 

Anyways, the only way I know of making this work is by changing the sender to some address on the Mailcow that is already created, and authenticating to it via username and password. I was hoping there might be an easier way where I could setup a few servers with a simple smart host for relaying emails that are packed with the proper DKIM and headers. 

Basically I have a web server, and I want it to send mail that are actually fully flushed out from my mailcow from multiple sending addresses without having to manually specify a username and password, and rewrite the headers of the email. 

If this isn't possible I'll just see if I can rewrite all emails to something like servers_noreply@example.com. I was just hoping there was an easier way that would allow for better delivery of my emails.  

Sorry for all the rambles above, and thank you for your time.

**Comment by Munzy** [comment]
https://github.com/mailcow/mailcow-dockerized/issues/2715 might be of some use.
